### DISCUSSION TITLE
more generation

### DISCUSSION THREAD

**Discussion by nalbion** [Pythagora-io/gpt-pilot#67] [Ideas]
There are some files which every project should have and are not generated by gpt-pilot:

- .gitignore
- .editorconfig
- README.md
- LICENSE  - ask me questions in simple terms to help me narrow down the options, and then explain the top candidates in simple terms.
- Dockerfile

- CI/CD pipelines & scripts - eg .github/workflows/ci.yml, scripts/test, scripts/deploy
- IaC - eg Cloudformation, serverless.yml
- company standard resource templates

Some projects may be better off initialised by create-react/next-app, Spring initialiser, Yeoman etc. A lot of these tools do not generate most of the files listed above.

**Comment by zvone187** [comment]
Hey @nalbion, yes, you're definitely correct and support for all of these should be added at one point. However, the idea is to get GPT Pilot to be able to work at any scale first (eg. that you can create an app and continue working on it indefinitely) and then start working on each of these. I'll keep this open since we'll start implementing these.

**Comment by MaxAnfilofyev** [comment]
related to https://github.com/Pythagora-io/gpt-pilot/discussions/899
